src/main/java/ru/practicum/shareit/request/mapper/ItemRequestMapper.java

```diff
@@ -0,0 +1,34 @@
+package ru.practicum.shareit.request.mapper;
+
+import ru.practicum.shareit.item.dto.ItemDto;
+import ru.practicum.shareit.request.dto.ItemRequestResponseDto;
+import ru.practicum.shareit.request.model.ItemRequest;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ItemRequestMapper {
+
+    private ItemRequestMapper() {
+    }
+
+    public static ItemRequestResponseDto toItemRequestResponseDto(ItemRequest itemRequest, List<ItemDto> itemDtos) {
+        return ItemRequestResponseDto.builder()
+                .id(itemRequest.getId())
+                .description(itemRequest.getDescription())
+                .created(itemRequest.getCreated())
+                .items(itemDtos)
+                .build();
+    }
+
+    public static List<ItemRequestResponseDto> toItemRequestResponseDto(List<ItemRequest> itemRequests, List<ItemDto> itemDtos) {
+        List<ItemRequestResponseDto> dtos = new ArrayList<>();
+        for (ItemRequest itemRequest : itemRequests) {
+            dtos.add(toItemRequestResponseDto(itemRequest,
+                    itemDtos.stream().filter(item -> item.getRequestId().equals(itemRequest.getId())).collect(Collectors.toList())
```

Comment:
Лучше разбить по строкам для лучшей читаемости
```suggestion
                    itemDtos.stream()
                        .filter(item -> item.getRequestId().equals(itemRequest.getId()))
                        .collect(Collectors.toList())
```